### PR TITLE
feat(switch): add attribute to invert the switch

### LINKED ISSUE #7889: md-switch doesn't have an option to align text
Right now the md-switch component only allows to have the text on the right side of the switch control
switch-right-side-text

It will be nice to have the ability to set the text on the left size of the switch control
switch-left-side-text

### PR DESCRIPTION
**Pull request by Lordnoname**
- Adds an attribute which invertes the switch when set to true

Fixes #7889

### CODE REVIEW

**Comment by Lordnoname** [comment]
I signed it!

**Comment by ThomasBurleson** [comment]
Thx @Lordnoname

**Comment by DevVersion** [comment]
@ThomasBurleson Yea, looks good to me.

**Comment by Lordnoname** [comment]
@ThomasBurleson Is this feature going to be implemented in release 1.1.0? I am asking, because you added the `deprecated` milestone despite having the `needs: merge` label applied.

**Comment by ThomasBurleson** [comment]
This issue is closed as part of our ‘Surge Focus on Material 2' efforts.
      For details, see our forum posting @ http://bit.ly/1UhZyWs.

**Comment by DevVersion** [comment]
@ThomasBurleson Ping on this PR, if you probably overlooked it.

**Comment by DevVersion** [comment]
@Lordnoname Can you please rebase that Pull Request?

**Comment by topherfangio** [comment]
@Lordnoname Ping for rebase.

**Comment by Lordnoname** [comment]
I am going to rebase asap. Im currently in vacations. :)

File: src/components/switch/switch.spec.js [hunk 1]
```diff
@@ -79,4 +79,20 @@ describe('<md-switch>', function() {
     expect(checkbox[0]).not.toHaveClass('md-checked');
 
   });
+
+  it('should correctly invert the switch through attribute', function() {
+    var element = $compile('<md-switch md-invert="{{ isInverted }}">')(parentScope);
+
+    parentScope.$apply('isInverted = true');
+
+    expect(element).toHaveClass('_md-inverted');
```
Comment (DevVersion) [inline src/components/switch/switch.spec.js]: Should be unprefixed (fixes the tests) - Same for below

File: src/components/switch/switch.scss [hunk 2]
```diff
@@ -48,6 +48,8 @@ md-switch {
     float: left;
   }
 
+
+
```
Comment (DevVersion) [inline src/components/switch/switch.scss]: Unnecessary lines can be removed

**Comment by topherfangio** [comment]
Just manually tested this, and it doesn't appear to actually invert the switch; just the text.

screen shot 2016-10-17 at 3 49 08 pm

Is that right? I would assume it should invert the actual control too?

**Pull request merged**